src/api-binder/report.ts

```diff
@@ -141,15 +142,30 @@ export async function startReporting() {
 	);
 	const doReport = async () => {
 		if (!reportPending) {
-			throttledReport(reportConfigs);
+			return throttledReport(reportConfigs);
```

Comment:
We don't care about the return value, only about awaiting it:
```suggestion
			await throttledReport(reportConfigs);
```